src/IJulia.jl

```diff
@@ -33,13 +33,16 @@ The `IJulia` module is used in three ways
 module IJulia
 export notebook, jupyterlab, installkernel
 
+import MbedTLS
 using ZMQ, JSON, SoftGlobalScope
-import Base.invokelatest
+import Base: invokelatest, RefValue
```

Comment:
Probably `using` rather than `import` since we only use these, not extend them?   Also `invokelatest` is exported automatically these days.

(This code dates back to the days before `using Foo: bar`)

Reply:
Agreed, but I think I'd prefer to do those sorts of changes in a later PR. This one is already quite large :sweat_smile: